docs/building.md

```diff
@@ -76,7 +76,9 @@ $ cd cpp-sc2
 $ cmake -B build
 
 # Build the project.
-$ cmake --build build --parallel
+$ cmake --build build --parallel $(nproc --ignore=0)
+
+# If building freezes in above step, increase number: --ignore=1, ignore=2 ...
```

Comment:
Lets reshape it in the following way:

```markdown
# Build the project.
cmake --build build --parallel $(nproc)

# If building freezes in above step, decrease nproc to 1 or more, e.g.:
cmake --build build --parallel $(nproc --ignore=1)
```

Reply:
Since the first command must work for everyone, we provide fallback command to those who have build issues.